docs/content/api.md

```diff
@@ -50,9 +50,9 @@ The animation will continue to run until the `mover` in the physics simulation h
 
 #### Example
 
-```typescript
-const GravityBasic: React.FC = () => {
-  const [props] = useGravity<HTMLDivElement>({
+```playground
+function GravityBasic() {
+  const [props] = useGravity({
     from: { opacity: 0 },
```

Comment:
I would recommend starting your API examples from 0.2 opacity rather than 0. I tried upping the r value in this example and thought at first that the playground was not responding 😨

Reply:
Yeah, `r` will probably trip folks up – the force increases as you reduce it (gravity becomes stronger when two bodies are closer together). But good call, I'll start the `opacity` at 0.25 or something like that.

In the long term I'd love to have something like `IntersectionObserver` running to mount the component when the user scrolls close to it, since the animation runs immediately on component mount and may finish before they even scroll to it. The other way to get around this is using `infinite: true` to make it animate continuously (which `react-spring` does on their site).